shub/deploy.py

```diff
@@ -128,14 +128,15 @@ def _url(endpoint, action):
 
 
 def _upload_egg(endpoint, eggpath, project, version, auth, verbose, keep_log,
-                stack=None, requirements_file=None):
+                stack=None, requirements_file=None, eggs=None):
+    eggs = eggs or []
     data = {'project': project, 'version': version}
     if stack:
         data['stack'] = stack
-    files = {'egg': ('project.egg', open(eggpath, 'rb'))}
+    files = [('egg', open(path, 'rb')) for path in eggs]
```

Comment:
The JIRA ticket says: "When deploying send [the eggs] in `eggs` field. The main egg should be still send in `egg` field". But now apparently we're sending everything in the `egg` field. Were the specs changed? How does Dash know which one is the project egg?